### PR TITLE
Fixes to the 1.19 version

### PR DESCRIPTION
**Pull request by moxiegirl**
Eliminating all kinds of cruft

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ page_keywords: API, Docker, rcli, REST, documentation
...
    /articles/basics/#bind-docker-to-another-hostport-or-a-unix-socket).
- - The API tends to be REST, but for some complex commands, like `attach`
-   or `pull`, the HTTP connection is hijacked to transport `STDOUT`,
-   `STDIN` and `STDERR`.
- - When the client API version is newer than the daemon's an HTTP
-   `400 Bad Request` error message is returned.
+ - The API tends to be REST. However, for some complex commands, like `attach`
+   or `pull`, the HTTP connection is hijacked to transport `stdout`,
+   `stdin` and `stderr`.
+ - When the client API version is newer than the daemon's, these calls return an HTTP
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: these calls or 'requests'?

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 2]
```diff
@@ -90,10 +90,10 @@ Query Parameters:
...
 -   **size** – 1/True/true or 0/False/false, Show the containers
         sizes
--   **filters** - a json encoded value of the filters (a map[string][]string) to process on the containers list. Available filters:
-  -   exited=&lt;int&gt; -- containers with exit code of &lt;int&gt;
-  -   status=(restarting|running|paused|exited)
-  -   label=`key` or `key=value` of a container label
+-   **filters** - a JSON encoded value of the filters (a `map[string][]string`) to process on the containers list. Available filters:
+  -   `exited=<int>`; -- containers with exit code of  `<int>` ;
+  -   `status=`(`restarting`|`running`|`paused`|`exited`)
+  -   `label=key` or `key=value` of a container label
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: We need more tables for things like this, but that's just a side-note :smile:

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 3]
```diff
@@ -109,131 +109,130 @@ Create a container
...
 Json Parameters:
 
--   **Hostname** - A string value containing the desired hostname to use for the
+-   **Hostname** - A string value containing the hostname to use for the
       container.
--   **Domainname** - A string value containing the desired domain name to use
+-   **Domainname** - A string value containing the domain name to use
       for the container.
--   **User** - A string value containing the user to use inside the container.
+-   **User** - A string value specifying the user to use inside the container.
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: "the user to use"

need something better at some point as well, no (short) suggestion at this moment. It's the user that the process inside the container will be run as.

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 4]
```diff
@@ -109,131 +109,130 @@ Create a container
...
 -   **Memory** - Memory limit in bytes.
--   **MemorySwap**- Total memory limit (memory + swap); set `-1` to disable swap,
-      always use this with `memory`, and make the value larger than `memory`.
--   **CpuShares** - An integer value containing the CPU Shares for container
+-   **MemorySwap**- Total memory limit (memory + swap); set `-1` to disable swap>
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: stray `>` after swap (was a comma)

File: docs/sources/reference/api/docker_remote_api_v1.19.md (hunk 3 above)
Comment (moxiegirl) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: went with `the user inside the container`

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 5]
```diff
@@ -109,131 +109,130 @@ Create a container
...
--   **Labels** - Adds a map of labels that to a container. To specify a map: `{"key":"value"[,"key2":"value2"]}`
+-   **Labels** - Adds a map of labels to a container. To specify a map: `{"key":"value"[,"key2":"value2"]}`
 -   **Cmd** - Command to run specified as a string or an array of strings.
--   **Entrypoint** - Set the entrypoint for the container a a string or an array
-      of strings
--   **Image** - String value containing the image name to use for the container
--   **Volumes** – An object mapping mountpoint paths (strings) inside the
+-   **Entrypoint** - Set the entry point for the container as a string or an array
+      of strings.
+-   **Image** - A string specifying the image name to use for the container.
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: "to use for", "to assign to"? just a mind blurb

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 6]
```diff
@@ -472,34 +471,34 @@ Status Codes:
...
+-   **follow** – 1/True/true or 0/False/false, return stream. Default `false`.
+-   **stdout** – 1/True/true or 0/False/false, show `stdout` log. Default `false`.
+-   **stderr** – 1/True/true or 0/False/false, show `stderr` log. Default `false`.
 -   **since** – UNIX timestamp (integer) to filter logs. Specifying a timestamp
     will only output log-entries since that timestamp. Default: 0 (unfiltered)
 -   **timestamps** – 1/True/true or 0/False/false, print timestamps for
-        every log line. Default false
--   **tail** – Output specified number of lines at the end of logs: `all` or `<number>`. Default all
+        every log line. Default `false`.
+-   **tail** – Output specified number of lines at the end of logs: `all` or `<number>`. Default all.
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: Default `all` (back-ticks)?

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 7]
```diff
@@ -974,17 +972,17 @@ Status Codes:
...
 
     The format is a **Header** and a **Payload** (frame).
 
     **HEADER**
 
-    The header will contain the information on which stream write the
-    stream (stdout or stderr). It also contain the size of the
-    associated frame encoded on the last 4 bytes (uint32).
+    The header contains the information which the stream writes (`stdout` or
+    `stderr`). It also contain the size of the associated frame encoded on the
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: s/contain/contains/

File: docs/sources/reference/api/docker_remote_api_v1.19.md (hunk 6 above)
Comment (moxiegirl) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: I don't think you have to specify all --- tail just does the right thing.  So, I'm gonna remove the ticks from the first `all` ping me if that is bad.

File: docs/sources/reference/api/docker_remote_api_v1.19.md (hunk 7 above)
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: "on the" or "in the"?

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 8]
```diff
@@ -974,17 +972,17 @@ Status Codes:
...
+    last 4 bytes (`uint32`).
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: s/4/four/

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 9]
```diff
@@ -974,17 +972,17 @@ Status Codes:
...
 
     It is encoded on the first 8 bytes like this:
```
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: s/8/eight/

(if the style guide is to be followed) :)

Comment (moxiegirl) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: Hmm, I think 8 bytes is part of a measurement.
image

Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: Right, good to know :+1:

Would have done that instinctively, but had "da guide" in mind

**Comment by thaJeztah** [comment]
Thanks! LGTM

**Comment by jfrazelle** [comment]
cool i will rebase after this ;)

**Pull request merged**